stor/dx.py

```diff
@@ -779,11 +779,11 @@ def upload(self, to_upload, **kwargs):
 
             if upload_obj.source.isfile():
                 dest_is_file = dest_file.isfile()
-                if dest_is_file:
-                    raise stor_exceptions.TargetExistsError(
+                if dest_is_file:  # only occurs if upload is called directly with existing objects
+                    print(stor_exceptions.TargetExistsError(
```

Comment:
Why do you create an instance of `TargetExistsError` here? Should this be a logger statement instead of print?

Reply:
yep that makes sense. changed to logger.warning. removed the TargetExistsError.